src/Apps/Order2/Routes/Checkout/Components/FulfillmentDetailsStep/Order2DeliveryForm.tsx

```diff
@@ -39,7 +39,7 @@ export const Order2DeliveryForm: React.FC<Order2DeliveryFormProps> = () => {
   ) => {}
   return (
     <>
-      <Text fontWeight="bold" color="mono100" variant="sm-display">
+      <Text fontWeight="400" color="mono100" variant={["sm-display", "md"]}>
```

Comment:
I think we want to keep words here for font weights - and are we intentionally making it less bold?

Edit: It looks like we want it to be `medium` at mobile and `normal` at desktop.